gsplat/cuda/csrc/rasterize_to_pixels_fwd.cu

```diff
@@ -53,6 +57,15 @@ __global__ void rasterize_to_pixels_fwd_kernel(
     bool inside = (i < image_height && j < image_width);
     bool done = !inside;
 
+    // when the mask is provided, render 0.0 and return if this tile
+    // is labeled as not local
+    if (masks != nullptr && inside && !masks[tile_id]) {
+        for (uint32_t k = 0; k < COLOR_DIM; ++k) {
+            render_colors[pix_id * COLOR_DIM + k] = 0.0f;
```

Comment:
If background is enabled, should set color as background

Reply:
fixed